pkg/enqueue/Consumption/QueueConsumer.php

```diff
@@ -195,7 +195,7 @@ public function consume(ExtensionInterface $runtimeExtension = null): void
             if (null === $result) {
                 try {
                     $result = $processor->process($message, $this->interopContext);
-                } catch (\Exception $e) {
+                } catch (\Exception | \Throwable $e) {
```

Comment:
not sure php 7.3 support it

Reply:
I think it is there since PHP 7.1 https://www.php.net/manual/en/migration71.new-features.php#migration71.new-features.mulit-catch-exception-handling

Reply:
Thanks for clarification